docs/conf.py

```diff
@@ -122,7 +122,7 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 html_theme_options = {
-    'canonical_url': "https://listenbrainz.readthedocs.io/en/production/",
+    'canonical_url': "https://listenbrainz.readthedocs.io/en/latest/",
```

Comment:
I'm not actually sure what this is used for, but based on the other changes here, I'm guessing this is correct (let me know if not, and I'll revert).

Reply:
Looks right